src/server/event_bus.c

```
@@ -0,0 +1,265 @@
+    }
+
+    remove_sub(e, fn);
+}
+
+obc_event_result_t obc_event_fire(const obc_engine_api_t *api,
+                                   const char             *event_name,
+                                   int                     sender_slot,
+                                   const void             *data)
+{
+    obc_event_result_t result = { false, false };
+    if (!validate_event_name(event_name, NULL)) return result;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)